modules/statics/src/networks.ts

```diff
@@ -405,6 +420,7 @@ export const Networks = {
     celo: Object.freeze(new CeloTestnet()),
     dash: Object.freeze(new DashTestnet()),
     eos: Object.freeze(new EosTestnet()),
+    ethereum2: Object.freeze(new Ethereum2Testnet()),
```

Comment:
should call it Pyrmont, as there will likely be more eth2 testnets that we need to support and differentiate